### PR TITLE
Fix deprecations via Deprecations.jl

### PR DESCRIPTION
**Pull request by ararslan**
`edit_file` run manually on all Julia files

### CODE REVIEW

File: src/codegen.jl [hunk 1]
```diff
@@ -62,59 +62,59 @@ end
...
 
-stripmodifier{f}(cppfunc::Type{CppFptr{f}}) = cppfunc
-stripmodifier{T,CVR}(p::Union{Type{CppPtr{T,CVR}},
-    Type{CppRef{T,CVR}}}) = p
-stripmodifier{T <: CppValue}(p::Type{T}) = p
-stripmodifier{T<:CppEnum}(p::Type{T}) = p
-stripmodifier{base,fptr}(p::Type{CppMFptr{base,fptr}}) = p
+stripmodifier(cppfunc::Type{CppFptr{f}}) where {f} = cppfunc
+stripmodifier(p::Union{Type{CppPtr{T,CVR}},
+    Type{CppRef{T,CVR}}}) where {T,CVR} = p
```
Comment (ararslan) [inline src/codegen.jl]: Something worth noting: I had to manually indent this, since `edit_file` lost the indent.

Comment (ararslan) [inline src/codegen.jl]: @Keno Relevant to Deprecations/FemtoCleaner I'd think? ^ I'd have "bad bot"ed but there was no bot.

**Comment by ararslan** [comment]
Bump @Keno

**Comment by Keno** [comment]
Thanks @ararslan, I appreciate it.

**Comment by ararslan** [comment]
My pleasure :)

**Pull request merged**